android/app/src/main/kotlin/com/example/flutter_module_11/MainActivity.kt

```diff
@@ -1,6 +1,14 @@
 package com.example.flutter_module_11
-
+import com.yandex.mapkit.MapKitFactory
 import io.flutter.embedding.android.FlutterActivity
+import io.flutter.plugins.GeneratedPluginRegistrant
+import io.flutter.embedding.engine.FlutterEngine
+
 
 class MainActivity: FlutterActivity() {
+
+    override fun configureFlutterEngine(flutterEngine: FlutterEngine) {
+        MapKitFactory.setApiKey("bcda53ad-8d47-49d0-86d2-4822abd4364e")
```

Comment:
Please store the API key in Android localProperties file. We should not expose secret key in remote code. But since it is for learning purposes - you can share this code in readme file saying that "If you want to launch the app - add line ... into your android localProperties file".

Reply:
done.